tools/python/util/vcpkg_helpers.py

```diff
@@ -367,66 +364,74 @@ def generate_vcpkg_triplets_for_emscripten(build_dir: str, emscripten_root: str)
         build_dir (str): The directory to save the generated triplet files.
         emscripten_root (str): The root path of Emscripten.
     """
-    for enable_rtti in [True, False]:
-        for enable_asan in [True, False]:
-            for target_abi in ["wasm32", "wasm64"]:
-                folder_name_parts = []
-                if enable_asan:
-                    folder_name_parts.append("asan")
-                folder_name = "default" if len(folder_name_parts) == 0 else "_".join(folder_name_parts)
-                os_name = "emscripten"
-                folder_name_parts = []
-                if enable_asan:
-                    folder_name_parts.append("asan")
-                if not enable_rtti:
-                    folder_name_parts.append("nortti")
-                folder_name = "default" if len(folder_name_parts) == 0 else "_".join(folder_name_parts)
-                file_name = f"{target_abi}-{os_name}.cmake"
-                dest_path = Path(build_dir) / folder_name / file_name
-                os.makedirs(dest_path.parent, exist_ok=True)
-                with open(dest_path, "w", encoding="utf-8") as f:
-                    add_copyright_header(f)
-                    f.write(r"""
+    for enable_wasm_exception_catching in [True, False]:
+        for enable_rtti in [True, False]:
+            for enable_asan in [True, False]:
+                for target_abi in ["wasm32", "wasm64"]:
+                    folder_name_parts = []
+                    if enable_asan:
+                        folder_name_parts.append("asan")
+                    folder_name = "default" if len(folder_name_parts) == 0 else "_".join(folder_name_parts)
+                    os_name = "emscripten"
+                    folder_name_parts = []
+                    if enable_asan:
+                        folder_name_parts.append("asan")
+                    if not enable_rtti:
+                        folder_name_parts.append("nortti")
+                    if enable_wasm_exception_catching:
+                        folder_name_parts.append("exception_catching")
```

Comment:
This should align with the build.py
```suggestion
                    if not enable_exception:
                        folder_name_parts.append("noexception")
                    elseif enable_wasm_exception_catching:
                        folder_name_parts.append("exception_catching")
```